website/content/docs/internals/integrated-storage.mdx

```diff
@@ -137,10 +137,12 @@ forwarded to the leader.
 Similar to other storage backends, data that is written to the Raft log and FSM
 will be encrypted by Vault's barrier.
 
-Vault does not currently offer automated dead server cleanup. If you wish to
-decommission a node, or a node dies and must be replaced, the node must manually
-be removed from the cluster with the `remove peer`
-[command](/docs/commands/operator/raft#remove-peer).
+To cleanup dead servers automatically, enable [dead server cleanup in the
+autopilot configuration](https://learn.hashicorp.com/tutorials/vault/raft-autopilot?in=vault/raft).
```

Comment:
```suggestion
autopilot configuration](/vault/tutorials/raft/raft-autopilot).
```

Reply:
Corrected to use `/vault/tutorials/raft/raft-autopilot` instead - thanks @tomhjp